code/modules/mob/living/simple_animal/abnormality/teth/scorched_girl.dm

```diff
@@ -55,6 +55,9 @@
 	else
 		patrol_path = get_path_to(src, target_center, /turf/proc/Distance_cardinal, 0, 200)
 
+/mob/living/simple_animal/hostile/abnormality/scorched_girl/MeleeAction()
```

Comment:
AttackingTarget() override is already added. You don't have to add this.

Reply:
Egor i saw it cause a runtime error.